.github/workflows/prod.yaml

```diff
@@ -54,8 +39,8 @@ jobs:
 
   howl-on-fail:
     runs-on: ubuntu-latest
-    needs: [ci, release]
-    if: ${{ always() && github.event_name == 'push' && ( needs.ci.result == 'failure' ||  needs.release.result == 'failure' ) }}
+    needs: [deploy-stage, release, deploy-prod]
+    if: failure()
```

Comment:
Are you sure this is correct? I think there was a reason why id didn't use it in the first place, but not sure anymore.... anyway, doesn't matter i have to rework this soon.

Reply:
I'm not completely sure - I thought it had something to do with some checks being skipped due to the conditional logic, but with the removal of the conditional jobs, I figured `failure()` is simpler.